.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,18 @@
+name: CVE2 bug
+description: Create a CVE2 bug
+title: "[BUG] <title>"
+labels: ["Type:Bug"]
+body:
+- type: checkboxes
```

Comment:
Do we want to continue with these "checkboxes"?  I am not sure they add a lot of value and a good percentage of Contributors simply ignore them.